coveo-pyproject/coveo_pyproject/commands.py

```diff
@@ -323,13 +323,14 @@ def refresh(project_name: str = None, exact_match: bool = False, verbose: bool =
 @pyproject.command()
 @click.argument("project_name", default=None, required=False)
 @click.option("--exact-match/--no-exact-match", default=False)
+@click.option("--fix/--no-fix", default=False)
 @click.option("--verbose", is_flag=True, default=False)
-def ci(project_name: str = None, exact_match: bool = False, verbose: bool = False) -> None:
+def ci(project_name: str = None, exact_match: bool = False, fix: bool = False, verbose: bool = False) -> None:
```

Comment:
You use the words `autofix` everywhere else. Why `fix` here?

Reply:
it's for the cli. That switch oughta be used mostly by developers before they commit. I though typing `--fix` was quick and nice, while typing `--auto-fix` was annoying (I had to retype it 3 times just now 😆 )